src/WebJobs.Script.WebHost/WebJobs.Script.WebHost.csproj

```diff
@@ -14,10 +14,10 @@
   <PropertyGroup Condition="'$(RuntimeIdentifier)' != ''">
     <PublishReadyToRun>true</PublishReadyToRun>
     <PublishReadyToRunUseCrossgen2>true</PublishReadyToRunUseCrossgen2>
-    <PublishReadyToRunComposite>false</PublishReadyToRunComposite> <!-- Setting to 'true' currently throws exception -->
+    <PublishReadyToRunComposite>true</PublishReadyToRunComposite>    
```

Comment:
Confirmed with @safihamid that we want this setting to true -- we'll gather data and see if it helps with cold start.